DSP Accel Library/Testing and Images/TestingLibraryScripts.m

```diff
@@ -0,0 +1,129 @@
+%% TESTING ONLY, USER DEFINED VARS
+clear;
+TableFn = "log(X_in)";
+maxInput = 1;
+floor_in_tab = 2^-12;
+error_cap_tab = 0.001;
+ERR_DIAG = true;
+N_bits_tab = 4;
+M_bits_tab = 5;
+igotthis = false;
+W_bits = 32;
+F_bits = 28;
```

Comment:
There're a couple different styles here...  Let's switch variables to camel case and constants to all caps to match the FE style guide